packages/edit-post/src/components/meta-boxes/index.js

```diff
@@ -50,7 +45,7 @@ export default function MetaBoxes( { location } ) {
 
 	return (
 		<>
-			{ map( metaBoxes, ( { id } ) => (
+			{ ( metaBoxes ?? [] ).map( ( { id } ) => (
```

Comment:
Could be nullish:

https://github.com/WordPress/gutenberg/blob/1cbecb49f1fefc0c5ac6bc5c6acbb7260a6f885b/packages/edit-post/src/store/selectors.js#L398